components/Workout/Workout.js

```diff
@@ -22,45 +25,78 @@ export default function Workout({ workout, handleDelete }) {
     setDeleteMode(!deleteMode);
   }
 
+  function toggleEditMode() {
+    setEditMode(!editMode);
+  }
+
+  function onSaveWorkout(name, currentExercises) {
+    handleEditWorkout(name, currentExercises, workout.id);
+  }
+
   return (
-    <WorkoutCard key={workout.id}>
-      {deleteMode ? (
-        <ModalOverlay onClick={toggleDeleteMode}>
-          <ModalContent onClick={(e) => e.stopPropagation()}>
-            <p>
-              Are you sure you want to delete <br />
-              <br /> &quot;{workout.name}&quot;?
-            </p>
-            <YesButton onClick={onDelete}>YES</YesButton>
-            <ModalButton onClick={toggleDeleteMode}>CANCEL</ModalButton>
-          </ModalContent>
-        </ModalOverlay>
+    <>
+      {editMode ? (
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            justifyContent: "center",
+          }}
+        >
+          <Filter onClick={toggleEditMode}></Filter>
+          <Form
+            formTitle="Edit your Workout"
+            toggleMode={toggleEditMode}
+            onSaveWorkout={onSaveWorkout}
+            workout={workout}
+            editMode={editMode}
+          />
+        </div>
       ) : null}
-      <DeleteWorkoutButton onClick={toggleDeleteMode}>–</DeleteWorkoutButton>
-      <h2>{workout.name}</h2>
-      <SpotlightHeading>Muscles In The Spotlight:</SpotlightHeading>
-      <MuscleGroupList>
-        {workout.muscleGroups.map((muscle) => (
-          <MuscleBadge key={muscle}>{muscle}</MuscleBadge>
-        ))}
-      </MuscleGroupList>
-      <ToggleButton onClick={() => toggleDetails(workout.id)}>
-        {isDetailsVisible ? "Show Less" : "Show More"}
-      </ToggleButton>
-      {isDetailsVisible && (
-        <ExerciseList>
-          {workout.exercises.map((exercise) => {
-            return (
-              <ExerciseItem key={exercise.exerciseId}>
-                <ExerciseName>{exercise.name}</ExerciseName>
-                <ExerciseDetails>Sets: {exercise.sets}</ExerciseDetails>
-                <ExerciseDetails>Reps: {exercise.reps}</ExerciseDetails>
-              </ExerciseItem>
-            );
-          })}
-        </ExerciseList>
-      )}
-    </WorkoutCard>
+      <WorkoutCard>
+        {deleteMode ? (
+          <ModalOverlay onClick={toggleDeleteMode}>
+            <ModalContent onClick={(e) => e.stopPropagation()}>
+              <p>
+                Are you sure you want to delete <br />
+                <br /> &quot;{workout.name}&quot;?
+              </p>
+              <YesButton onClick={onDelete}>YES</YesButton>
+              <ModalButton onClick={toggleDeleteMode}>CANCEL</ModalButton>
+            </ModalContent>
+          </ModalOverlay>
+        ) : null}
+
```

Comment:
I think here instead of using a ternary with null, you can adjust your code  and just use the operator &&
for example:` {deleteMode && (
          <ModalOverlay onClick={toggleDeleteMode}>
            <ModalContent onClick={(e) => e.stopPropagation()}>
              <p>
                Are you sure you want to delete <br />
                <br /> &quot;{workout.name}&quot;?
              </p>
              <YesButton onClick={onDelete}>YES</YesButton>
              <ModalButton onClick={toggleDeleteMode}>CANCEL</ModalButton>
            </ModalContent>
          </ModalOverlay>
        )}`